apis/networking/v1/gcpbackendpolicy_types.go

```diff
@@ -50,7 +50,7 @@ type GCPBackendPolicyList struct {
 // GCPBackendPolicySpec defines the desired state of GCPBackendPolicy.
 type GCPBackendPolicySpec struct {
 	// TargetRef identifies an API object to apply policy to.
-	TargetRef v1alpha2.PolicyTargetReference `json:"targetRef"`
+	TargetRef v1alpha2.NamespacedPolicyTargetReference `json:"targetRef"`
```

Comment:
Please make a corresponding change to update our sources with the new type

Reply:
Done.